TombEngine/Game/items.cpp

```diff
@@ -133,14 +177,14 @@ void KillItem(short const itemNumber)
 
 		g_GameScriptEntities->NotifyKilled(item);
 		g_GameScriptEntities->TryRemoveColliding(itemNumber, true);
-		if (!item->LuaCallbackOnKilledName.empty())
-			g_GameScript->ExecuteFunction(item->LuaCallbackOnKilledName, itemNumber);
+		if (!item->Lua.CallbackOnKilledName.empty())
```

Comment:
Should be part of [a5e602d](https://github.com/MontyTRC89/TombEngine/pull/828/commits/a5e602dc40d8f0f31b59e7c92176d2a6ec48dde1)